docs/streams/upgrade-guide.html

```diff
@@ -152,6 +152,12 @@ <h3><a id="streams_api_changes_390" href="#streams_api_changes_390">Streams API
         The specified handler must implement the <code>org.apache.kafka.streams.errors.ProcessingExceptionHandler</code> interface.
     </p>
 
+    <p>
+        Kafka Streams now supports users to customize the output interval of stream-thread runtime summary.
+        By default, it logs output every 2 minutes. More details can be found in
```

Comment:
```suggestion
        By default, the summary is logged every 2 minutes. More details can be found in
```

Reply:
Thanks for your reply, I have made the changes